### PR TITLE
Is admin policy

### PR DESCRIPTION
**Pull request by GALTdea**
## What's the change?
Adds a general reusable headless user_only_policy to be used across the application

Please delete items that are not relevant.

- [x] Did you add appropriate automated tests?
- [x] Did you consider risks around security, performance, etc.?
- [x] Is there any linting that needs to be executed?
- [x] Did you leave helpful inline PR comments for the reviewer(s)?

### CODE REVIEW

**Comment by GALTdea** [comment]
This PR will help complete the #225 issue by adding a  policy method that will help hide notes and any links from the user_mentee_application show page. I think I'll wait until we merge the PR we're working on to merge this one

**Review by toyhammered — APPROVED** [review]

**Pull request merged**
